test/ui-testing/pick-lists.js

```diff
@@ -0,0 +1,268 @@
+              document,
+              null,
+              XPathResult.FIRST_ORDERED_NODE_TYPE
+            ).singleNodeValue;
+            if (row != null) {
+              throw Error(`Should not find row with list ${list}`);
+            }
+          }, pickList)
+          .then(done)
+          .catch(done);
+      });
+
+
+      it(`should create pick list value ${pickListValue} in settings`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-list-values"]')
+          .click('a[href="/settings/licenses/pick-list-values"]')
+          .waitUntilNetworkIdle(1000)
+          .wait('#categorySelect')
+          .type('#categorySelect', editedPickList)
+          .wait('#clickable-add-pick-list-values')
+          .click('#clickable-add-pick-list-values')
+          .wait('input[name="items[0].label"]')
+          .type('input[name="items[0].label"]', pickListValue)
+          .wait('#clickable-save-pick-list-values-0')
+          .click('#clickable-save-pick-list-values-0')
+          .wait(222)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should find pick list value ${pickListValue} in list`, done => {
+        nightmare
+          .wait('#editList-pick-list-values')
+          .evaluate(value => {
+            const valueRows = [...document.querySelectorAll('#editList-pick-list-values')].map(e => e.textContent);
+            const valueElement = valueRows.find(r => r.indexOf(value) >= 0);
+            if (!valueElement) {
+              throw Error(`Could not find row with the edited value ${value} but found ${valueElement}`);
+            }
+          }, pickListValue)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should edit pick list value ${pickListValue} in settings`, done => {
+        nightmare
+          .wait('a[href="/settings/licenses"]')
+          .click('a[href="/settings/licenses"]')
+          .wait('a[href="/settings/licenses/pick-list-values"]')
+          .click('a[href="/settings/licenses/pick-list-values"]')
+          .waitUntilNetworkIdle(1000)
+          .wait('#categorySelect')
+          .type('#categorySelect', editedPickList)
+          .wait('#clickable-edit-pick-list-values-0')
+          .click('#clickable-edit-pick-list-values-0')
+          .wait('input[name="items[0].label"]')
+          .type('input[name="items[0].label"]', editText)
+          .wait('#clickable-save-pick-list-values-0')
+          .click('#clickable-save-pick-list-values-0')
+          .wait(222)
+          .then(done)
+          .catch(done);
+      });
+
+      it(`should find edited pick list value ${editedPickListValue} in list`, done => {
+        nightmare
+          .wait('#editList-pick-list-values')
+          .evaluate(value => {
+            const valueRows = [...document.querySelectorAll('#editList-pick-list-values')].map(e => e.textContent);
```

Comment:
I don't think this `evaluate` is working as intended. The `querySelectorAll` is just going to return the single element for the item by the ID `editList-pick-lists`.

You could try `[...document.querySelectorAll('#editList-pick-list-values [role="row"]')]` to fetch the _rows_ rather than the list itself.